system-parachains/bridge-hubs/bridge-hub-kusama/primitives/src/lib.rs

```diff
@@ -87,7 +98,7 @@ frame_support::parameter_types! {
 	/// The XCM fee that is paid for executing XCM program (with `ExportMessage` instruction) at the Kusama
 	/// BridgeHub.
 	/// (initially was calculated by test `BridgeHubKusama::can_calculate_weight_for_paid_export_message_with_reserve_transfer` + `33%`)
-	pub const BridgeHubKusamaBaseXcmFeeInKsms: u128 = 16_156_041_984;
+	pub const BridgeHubKusamaBaseXcmFeeInKsms: u128 = 386_365_000;
```

Comment:
Cannot be calculated on flight and be non const?

Reply:
It cannot because those constants/fees are used in AssetHub runtimes and the calculation is based on the real BridgeHub's `WeightToFee` and `XcmConfig::Weigher`.